### ISSUE TITLE
ES Module Error

### ISSUE THREAD

**Issue by thoril-pegason** [PsychoLlama/media-devices#436]
Given this `package.json`:
```json
{
  "type": "module",
  "dependencies": {
    "media-devices": "^0.4.0"
  }
}
```

And this `index.js`:
```javascript
import MediaDevices from 'media-devices'
```
Running `node index.js` gives the following error:

```
Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
/home/t/r88/proboscis/more/node_modules/media-devices/dist/media-devices.es.js:131
export { DeviceKind, OperationType, index as default, supportsMediaDevices };
^^^^^^

SyntaxError: Unexpected token 'export'
 (stack trace)
Node.js v20.8.0
```

**Comment by PsychoLlama** [comment]
Hey, thanks for the report. I'm able to repro on my side, too.

I've made a few changes to the build system recently that should fix this (staged in `0.4.1-rc.1`). I'll clean those up and publish  `0.5`.

**Comment by PsychoLlama** [comment]
This should be fixed in [v0.5.0](https://github.com/PsychoLlama/media-devices/releases/tag/v0.5.0). Closing the ticket, but let me know if it gives you any more trouble. Thanks again for the report!
